applications/FluidDynamicsApplication/tests/TwoFluidMassConservationSourceTest/ProjectParameter3D.json

```diff
@@ -37,6 +37,11 @@
             "dynamic_tau": 1.0,
             "mass_source": true
         },
+        "levelset_convection_settings": {
+            "element_settings" : {
+                "dynamic_tau" : 1.0
+            }
+        },
```

Comment:
Does it crash without adding this?

Reply:
it does not crash but some tests were failing. The default value is 0.0 here but before the changes of this PR it was actually using the dynamic_tau from the two fluids solver (1.0), so in order to avoid changing the json results, I changed the tests so they use 1.0 here instead of the default value.

Reply:
OK. Perfect then. I was afraid we were not properly checking the defaults.